src/clients/c++/perf_client/perf_client.cc

```diff
@@ -259,142 +275,293 @@ Usage(char** argv, const std::string& msg = std::string())
+                   "required for sending the input in a batch-1 request. The "
+                   "client will reuse the data to match the specified batch "
+                   "size. Note that the files should contain only the raw "
+                   "binary representation of the data in row major order. By "
+                   "default, the client will use random data in its requests.",
+                   18)
+            << std::endl;
+  std::cerr << FormatMessage(
+                   " --shape: The shape used for the specified input. The "
+                   "argument must be specified as 'name:shape' where the shape "
+                   "is a comma-separated list for dimension sizes, for example "
+                   "'--shape input_name:1,2,3'. --shape may be specified "
+                   "multiple times to specify shapes for different inputs.",
+                   18)
+            << std::endl;
+  std::cerr << FormatMessage(
+                   " --sequence-length: It indicates the base length of a "
+                   "sequence used for sequence models. A sequence with length "
+                   "x will be composed of x requests to be sent as the "
+                   "elements in the sequence. The length of the actual "
+                   "sequence will be within +/- 20% of the base length.",
+                   18)
+            << std::endl;
+  std::cerr << "\tDEPRECATED OPTIONS" << std::endl;
+  std::cerr << std::setw(9) << std::left << " -z: "
+            << FormatMessage(
+                   "This flag causes input tensors to be initialized with "
+                   "zeros instead of random data",
+                   9)
             << std::endl;
   std::cerr
-      << "For -H, the header will be added to HTTP requests (ignored for GRPC "
-         "requests). The header must be specified as 'Header:Value'. -H may be "
-         "specified multiple times to add multiple headers."
+      << FormatMessage(
+             " --data-directory: It indicates that the perf client will use "
+             "user provided data instead of synthetic data for model inputs. "
+             "There must be a binary file for each input required by the "
+             "model. The file must be named the same as the input and must "
+             "contain data required for sending the input in a batch-1 "
+             "request. The perf client will reuse the data to match the "
+             "specified batch size. Note that the files should contain only "
+             "the raw binary representation of the data in row major order.",
+             18)
       << std::endl;
-  std::cerr << "The -z flag causes input tensors to be initialized with zeros "
-               "instead of random data"
+  std::cerr << std::endl;
+  std::cerr << "III. SERVER DETAILS: " << std::endl;
+  std::cerr << std::setw(9) << std::left << " -u: "
+            << FormatMessage(
+                   "This option is used to specify URL to the server. Default "
```

Comment:
Using "it" or "this option" is redundant. Just say "Specify the URL ..."
Update to show the default for both http and grpc protocols